aten/src/ATen/CMakeLists.txt

```diff
@@ -130,8 +130,8 @@ file(GLOB native_cuda_h "native/cuda/*.h" "native/cuda/*.cuh")
 file(GLOB native_cuda_linalg_cpp "native/cuda/linalg/*.cpp")
 file(GLOB native_hip_h "native/hip/*.h" "native/hip/*.cuh")
 file(GLOB native_cudnn_cpp "native/cudnn/*.cpp")
-file(GLOB native_nested_cuda_cu "native/nested/cuda/*.cu")
-file(GLOB native_nested_cuda_cpp "native/nested/cuda/*.cpp")
+file(GLOB native_nested_cuda_cu "native/nested/cuda/*.cu" "native/nested/flash_attn/*.cu")
+file(GLOB native_nested_cuda_cpp "native/nested/cuda/*.cpp" "native/nested/flash_attn/*.cpp")
 file(GLOB native_sparse_cuda_cu "native/sparse/cuda/*.cu")
```

Comment:
This took too long to find, but glad I know why arbitrarily adding folders doesn't work in native